src/bci_build/package/__init__.py

```diff
@@ -118,6 +118,51 @@ def __str__(self) -> str:
         return self.value
 
 
+@enum.unique
+class Registry(enum.Enum):
+    """Supported registries for container image distribution"""
+
+    SUSE_COM = "registry.suse.com"
+    APP_COLL = "dp.apps.rancher.io"
+    OPENSUSE_ORG = "registry.opensuse.org"
```

Comment:
why  do we need the  registry as enumerators?  are you enumerating (aka going over each one of them) anywhere? seems not the ideal abstraction to me.

Reply:
This IMHO would look a bit better:

```python
from collections import namedtuple

RegistryInfo = namedtuple("RegistryInfo", ["title", "prefix", "devel_prefix"])

class Registry:
    SUSE = RegistryInfo(title="SUSE Registry", prefix="suse", devel_prefix="bci")
    RANCHER = RegistryInfo(title="Rancher Prime Application Collection", prefix="containers", devel_prefix="containers")
    OPENSUSE = RegistryInfo(title="openSUSE Registry", prefix="opensuse", devel_prefix="opensuse/bci")

print(f"{Registry.SUSE.title} - {Registry.SUSE.prefix}")
print(f"{Registry.RANCHER.title} - {Registry.RANCHER.prefix}")
print(f"{Registry.OPENSUSE.title} - {Registry.OPENSUSE.prefix}")
```